Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.21 AS builder
+FROM registry.access.redhat.com/ubi9/go-toolset:1.20.10-2 AS builder
```

Comment:
We need Go 1.21.x

Reply:
We don't have redhat Go 1.21.x image yet. The latest one 1.20.10-3.